components/Footer.tsx

```diff
@@ -0,0 +1,139 @@
+        <div
+          className="col-auto max-w-2xl flex flex-col sm:flex-row sm:col-span-3 md:col-span-2 md:flex-col lg:flex-row items-center"
+          style={{ maxWidth: "25rem" }}
+        >
+          <img className="h-60 m-auto" src={footerLogo} alt="Footer Logo" />
+          <div style={{ width: "20rem" }}>
+            <p
+              className="text-neutral-300 text-center sm:text-left md:text-center lg:text-left"
+              id="desc"
+            >
+              Advisory is a fully youth-led non-profit dedicated to empowering
+              young Singaporeans to make informed decisions about their futures.
+            </p>
+          </div>
+        </div>
+        <div
+          className="text-center sm:text-left md:text-center lg:text-left"
+          id="grid1"
+        >
+          <h4 className="text-white">WHO WE ARE</h4>
+          <p className="page-links">
+            <a className="text-neutral-300" href="/team/">
+              Our Team
+            </a>
+          </p>
+          <p className="page-links">
+            <a className="text-neutral-300" href="/our-partners/">
+              Our Partners
+            </a>
+          </p>
+          <p className="page-links">
+            <a className="text-neutral-300" href="/about-us/">
+              About Us
+            </a>
+          </p>
+        </div>
+        <div className="text-center sm:text-left md:text-center lg:text-left">
+          <h4 className="text-white">WHAT WE DO</h4>
+          <p className="page-links">
+            <a className="text-neutral-300" href="/events/">
+              Our Events
+            </a>
+          </p>
+          <p className="page-links">
+            <a className="text-neutral-300" href="/stories/">
+              Our Stories
+            </a>
+          </p>
+          <p className="page-links">
+            <a className="text-neutral-300" href="/impact/">
+              Our Impact
+            </a>
+          </p>
+        </div>
+        <div className="text-center sm:text-left md:text-center lg:text-left">
+          <h4 className="text-white">GET INVOLVED</h4>
+          <p className="page-links">
+            <a className="text-neutral-300" href="/contact-us/">
+              Contact Us
+            </a>
+          </p>
+          <p className="page-links">
+            <a className="text-neutral-300" href="/join-us/">
+              Join Us
+            </a>
+          </p>
+        </div>
+      </div>
+      <div className="w-screen max-w-screen-lg mx-auto grid grid-cols-1 sm:grid-cols-2 px-16 py-8 justify-center sm:justify-between">
+        <div className="justify-self-center sm:justify-self-start">
+          <p className="text-neutral-300">© 2022 Advisory SG</p>
+        </div>
+        <div className="justify-self-center sm:justify-self-end flex flex-row gap-4">
+          {/* Replace these placeholder social media icons with your actual icons or images */}
```

Comment:
Is it possible to use the SVG icons from https://github.com/AdvisorySG/dawn-advisory-theme/blob/main/partials/icons.hbs?